sycl/doc/design/CompileTimeProperties.md

```diff
@@ -515,12 +515,56 @@ The front-end encodes the properties from the C++ attribute
 * The third parameter is the name of the source file (as with any call to this
   intrinsic).
 * The fourth parameter is the line number (as with any call to this intrinsic).
-* The fifth parameter is a metadata tuple with information about all of the
-  properties.  The first element of the tuple is a string literal with the name
-  of the first property.  The second element is the value of the first
-  property.  The third element is a string literal with the name of the second
-  property, etc.  Since each property has exactly one value, this tuple has an
-  even number of elements.
+* The fifth parameter is a pointer to a constant global variable. The type of
+  this variable is an anonymous structure. The first field of the structure is
+  a pointer to a string literal representing the name of the first property. The
+  second field of the structure is a pointer to a string literal representing
+  the value of the first property. The third field of the structure is a pointer
+  to a string literal representing the name of the second property, etc.
+  Since each property has exactly one value, this tuple has an even number of
+  elements. Pointers to property value strings may be a null-pointer, signalling
+  a property without a value.
+
+The resulting LLVM IR for the previous example would be:
```

Comment:
Is the LLVM IR listing below supposed to replace the LLVM IR listing above (starting on line 473)?  It seems confusing to have both listings.  What does the first listing mean?

Reply:
I originally kept the previous example as a "how it would look if we used `clang::annotate`", but since the two examples are so similar it will likely confuse more than it explains. I have removed the first example.

Reply:
Yes, this seems less confusing now.  Thanks.